airbyte-integrations/connectors/source-zendesk-talk/unit_tests/test_streams.py

```diff
@@ -233,3 +234,50 @@ def test_parse_response(self, mocker, now):
         result = list(stream.parse_response(response=response))
 
         assert result == [{"field1": "value", "field2": 3, stream.primary_key: int(now().timestamp())}]
+
+
+class TestIVRMenusStream:
+    def test_ivr_menus_parse_response(self, mocker):
+        stream = IVRMenus(subdomain="stest-domain", authenticator=mocker.Mock())
+        ivrs = [
+            {"id": random.randint(10000, 99999), "menus": [dict(key="value")]},
+            {"id": random.randint(10000, 99999), "menus": [dict(key="value")]},
+            {"id": random.randint(10000, 99999), "menus": [dict(key="value")]},
+            {"id": random.randint(10000, 99999), "menus": [dict(key="value")]},
+        ]
+        response_data = {
+            "ivrs": ivrs
+        }
+        response = MagicMock()
+        response.json.side_effect = [response_data]
+        for i, menu in enumerate(stream.parse_response(response)):
```

Comment:
An assertion on the number of returned records could help to make sure no record are dropped.

Reply:
Done